include/Mat.h

```diff
@@ -359,7 +373,7 @@ class Mat {
     }
 
     Mat<bool> operator&(bool b){
```

Comment:
open a ticket to evaluate the difference between the python behavior and c++
a=np.array([2,4])
b=np.array([1,3])
print(a|b)